utils/getInvoices/tests/getInvoices.placing.test.ts

```diff
@@ -5,139 +5,892 @@ import { expect } from '@jest/globals'
-    const payment: Partial<IPayment> = {
-      invoice: [{ type: ServiceType.Custom, price: 0, sum: 100 }],
-    }
-    const prevPayment: Partial<IPayment> = null
+  describe('props: { service, company, payment } with prio to payment', () => {
+    it('should NOT load when service = null, company = null, payment = null', () => {
+      const service: Partial<IService> = null
+      const company: Partial<IRealestate> = null
+      const payment: Partial<IPayment> = null
+
+      const invoices = getInvoices({
+        service,
+        company,
+        payment,
+      })
 
-    const invoices = getInvoices({
-      company,
-      service,
-      payment,
-      prevPayment,
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.Placing})
+      )
     })
 
-    expect(invoices).not.toContainEqual(
-      expect.objectContaining({ type: ServiceType.Placing })
-    )
+    it('should NOT load when service = null, company = null, payment = { invoice: [Inflicion] }', () => {
+      const service: Partial<IService> = null
+      const company: Partial<IRealestate> = null
+      const payment: Partial<IPayment> = {
+        invoice: [
+          {
+            type: ServiceType.Inflicion,
+            price: 15,
+            sum: 11,
+          },
+        ],
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+        payment,
+      })
+
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.Placing})
+      )
+    })
+
+    it('should load when service = null, company = null, payment = { invoice: [Placing] }', () => {
+      const service: Partial<IService> = null
+      const company: Partial<IRealestate> = null
+      const payment: Partial<IPayment> = {
+        invoice: [
+          {
+            type: ServiceType.Placing,
+            amount: 17,
+            price: 15,
+            sum: 11,
+          },
+        ],
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+        payment,
+      })
+
+      expect(invoices).toContainEqual({
+        type: ServiceType.Placing,
+        amount: 17,
+        price: 15,
+        sum: 11,
+      })
+    })
+
+    it('should load when service = { inflicionPrice: 10 }, company = null, payment = { invoice: [Placing] }', () => {
+      const service: Partial<IService> = {
+        inflicionPrice: 10,
+      }
+      const company: Partial<IRealestate> = null
+      const payment: Partial<IPayment> = {
+        invoice: [
+          {
+            type: ServiceType.Placing,
+            amount: 17,
+            price: 15,
+            sum: 13,
+          },
+        ],
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
+        payment,
+      })
+
+      expect(invoices).toContainEqual({
+        type: ServiceType.Placing,
+        amount: 17,
+        price: 15,
+        sum: 13,
+      })
+    })
```

Comment:
add some test-cases where we have valid data in both creating ways:
- valid `company` and `service` data
- valid `payment.invoice` data

check prio to payment